simplify_test.go

```diff
@@ -0,0 +1,34 @@
+package irutil
+
+import (
+	"testing"
+
+	"github.com/llir/llvm/ir/constant"
+	"github.com/llir/llvm/ir/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
To limit the number of external dependencies (see llir/llvm#92), and to have constant handling of test cases, I'd prefer if we try to rely on the official test framework of Go (`testing`). However, as this pertains to test cases and not the core project, I could be persuaded otherwise. What are the main benefits (and drawbacks) with using external test libraries as opposed to using the standard Go library for testing?

Reply:
Then we don't have to implement `expected xxx, actual yyy` everywhere

Reply:
Given that it is test code, I'll be fine with adding a few extra dependencies. So long as we keep the dependencies of the core packages to a minimum.